Fakebook.Posts/Fakebook.Posts.DataAccess/Models/FakebookPostsContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace Fakebook.Posts.DataAccess.Models {
+    public partial class FakebookPostsContext : DbContext {
+        public FakebookPostsContext(DbContextOptions<FakebookPostsContext> options) : base(options) { }
+        public virtual DbSet<Post> Posts { get; set; }
+        public virtual DbSet<Comment> Comments { get; set; }
+        protected override void OnModelCreating(ModelBuilder modelBuilder) {
+            modelBuilder.Entity<Post>(entity => {
+                entity.ToTable("Post");
+                entity.Property(e => e.CreatedAt).HasDefaultValueSql("NOW()").ValueGeneratedOnAdd();
+                entity.Property(e => e.Picture).HasDefaultValue("").ValueGeneratedOnAddOrUpdate();
```

Comment:
why is picture ValueGeneratedOnAddOrUpdate? that's not right.
and the default should just be NULL (as it is by default), shouldn't it?

Reply:
content and useremail should be required too... these things should not be being overlooked.
also, switch the DateTime properties to [DateTimeOffset](https://docs.microsoft.com/en-us/dotnet/standard/datetime/choosing-between-datetime#the-datetimeoffset-structure), so we don't have time zone problems.

Reply:
Property specifications have now been updated.